app/src/main/java/com/materiiapps/gloom/ui/GloomActivity.kt

```diff
@@ -51,60 +41,37 @@ class GloomActivity : ComponentActivity() {
         installSplashScreen()
         super.onCreate(savedInstanceState)
 
-        WindowCompat.setDecorFitsSystemWindows(window, false)
         getKoin().declare<Context>(this@GloomActivity)
 
         setContent {
-            val isDark = when (prefs.theme) {
-                Theme.SYSTEM -> isSystemInDarkTheme()
-                Theme.LIGHT -> false
-                Theme.DARK -> true
-            }
-
-            GloomTheme(isDark, prefs.monet) {
-                val systemUiController = rememberSystemUiController()
-                val navNarOffset = -((80 + 24).dp.toPx())
-                val defaultScreen = if (auth.isSignedIn)
-                    RootScreen()
-                else
-                    LandingScreen()
+            val navNarOffset = -((80 + 24).dp.toPx()) // Roughly the height of a navbar
```

Comment:
navNar :trolley:

Reply:
oops

Reply:
Also, why hardcode the offset?

Reply:
afaik i can't dynamically calculate it, i got to the 24dp number through trial and error.

i do plan on looking for a better solution soon though